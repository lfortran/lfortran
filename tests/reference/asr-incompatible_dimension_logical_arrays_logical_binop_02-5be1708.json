{
    "basename": "asr-incompatible_dimension_logical_arrays_logical_binop_02-5be1708",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/incompatible_dimension_logical_arrays_logical_binop_02.f90",
    "infile_hash": "c26b31652036dcebc000ad9bfe9d5aae3fac2441958efe2268f46d19",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-incompatible_dimension_logical_arrays_logical_binop_02-5be1708.stderr",
    "stderr_hash": "51654ae3483b5cb57d2295f9bf6b7501f2f1f8d1250b8a09f4c5e924",
    "returncode": 2
}