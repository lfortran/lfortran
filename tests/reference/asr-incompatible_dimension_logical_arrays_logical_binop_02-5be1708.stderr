semantic error: Shapes for operands are not conformable
 --> tests/errors/incompatible_dimension_logical_arrays_logical_binop_02.f90:4:14
  |
4 |     print *, x1 .neqv. x2
  |              ^^        ^^ 
